### ISSUE TITLE
Is it allowed by google play & app store to track users location?

### ISSUE THREAD

**Issue by feritzcan2** [transistorsoft/react-native-background-geolocation#2027]
Hello, we are developing an app that tracks user location and save on the cloud. 
The aim of the application is to record which places the user walked & which attractions he saw and show them on map..

But to persist data for user, we need to save into a database.

Is it against rules to save location of user?

**Comment by christocracy** [comment]
No, it’s not against the rules, you just have to tell them exactly what you’re doing and provide a privacy policy stating you’re not selling that information to 3rd parties.
